features/comments/commenting.feature

```diff
@@ -5,7 +5,11 @@ Feature: Commenting
   I want to store and view comments on a resource
 
   Background:
-    Given a post with the title "Hello World" written by "Jane Doe" exists
+    Given a configuration of:
+    """
+      ActiveAdmin.application.namespace(:admin).authorization_adapter = ActiveAdmin::AuthorizationAdapter
+    """
+    And a post with the title "Hello World" written by "Jane Doe" exists
```

Comment:
You can leave this setup untouched and instead tag each of the new specs with the [`@authorization` tag](https://github.com/activeadmin/activeadmin/blob/9292dfb548b7779db962ab7992ea4b1342ea69d4/features/support/env.rb#L110-L115).